meson.build

```diff
@@ -7,12 +7,16 @@ project(
 	meson_version: '>=0.43.0',
 	default_options: [
 		'cpp_std=c++14',
-        'c_std=c11',
+		'c_std=c11',
 		'warning_level=2',
 		'werror=false',
+		'sysconfdir=/etc',
 	],
 )
 
+prefixdir=get_option('prefix')
+sysconfdir=join_paths(prefixdir, get_option('sysconfdir'))
```

Comment:
hm, I am not sure about this one. This will be something like `/usr/etc` which is a pretty weird place to put config into. Maybe just `sysconfdir`?

Reply:
You're saying check if the prefix is the default of /usr/local and if so, use /etc/wayfire/ for the config directory, otherwise, use $prefix/etc/wayfire/? This doesn't sound right. For starters, how do you determine if --prefix=/usr/local was passed or not?